provisioner/kuberpak/controllers/bundle_controller.go

```diff
@@ -95,19 +95,21 @@ func (r *BundleReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ c
 	pod := &corev1.Pod{}
 	if op, err := r.ensureUnpackPod(ctx, bundle, pod); err != nil {
 		u.UpdateStatus(updater.SetBundleInfo(nil), updater.EnsureBundleDigest(""))
-		return ctrl.Result{}, updateStatusUnpackFailing(&u, fmt.Errorf("ensure unpack pod: %v", err))
+		return ctrl.Result{}, updateStatusUnpackFailing(&u, fmt.Errorf("ensure unpack pod: %w", err))
 	} else if op == controllerutil.OperationResultCreated || op == controllerutil.OperationResultUpdated || pod.DeletionTimestamp != nil {
 		updateStatusUnpackPending(&u)
 		return ctrl.Result{}, nil
 	}
 
 	switch phase := pod.Status.Phase; phase {
 	case corev1.PodPending:
-		return ctrl.Result{}, r.handlePendingPod(&u, pod)
+		r.handlePendingPod(&u, pod)
```

Comment:
Hmm interesting - this is another case where golangci-lint linting surprises me. Any idea on which specific linter was complaining about this implementation?

Reply:
This was `unparam` with this message:

> unparam      (*BundleReconciler).handlePendingPod - result 0 (error) is always nil

I also thought it was quite strange.

Reply:
Yeah I've had issues with this linter in the past, but it's possible it's uncovering a true error? In any case, I think it's fine to keep it enabled, and we can revisit reverting back to the original implementation once we get a better testing surface.

Reply:
Oh I realize it's because we only return nil errors from that function, which means the error return variable can be dropped, which makes more sense now.